File: selector/src/main/java/com/luck/picture/lib/interfaces/OnPlayerListener.java
package com.luck.picture.lib.interfaces;

/**
 * @author：luck
 * @date：2022/7/1 23:25 下午
 * @describe：OnPlayerListener
 */
public interface OnPlayerListener {
    /**
     * player error
     */
    void onPlayerError();

    /**
     * playing
     */
    void onPlayerReady();

    /**
     * preparing to play
     */
    void onPlayerLoading();

    /**
     * end of playback
     */
    void onPlayerEnd();
}

File: selector/src/main/java/com/luck/picture/lib/interfaces/OnPlayerListener.kt
package com.luck.picture.lib.interfaces

/**
 * @author：luck
 * @date：2022/7/1 23:25 下午
 * @describe：OnPlayerListener
 */
interface OnPlayerListener {
    /**
     * player error
     */
    fun onPlayerError()

    /**
     * playing
     */
    fun onPlayerReady()

    /**
     * preparing to play
     */
    fun onPlayerLoading()

    /**
     * end of playback
     */
    fun onPlayerComplete()
}